src/main/java/com/minecraftabnormals/abnormals_core/core/api/conditions/config_predicates/GreaterThanOrEqualPredicate.java

```diff
@@ -0,0 +1,59 @@
+package com.minecraftabnormals.abnormals_core.core.api.conditions.config_predicates;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+import com.minecraftabnormals.abnormals_core.core.AbnormalsCore;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.ForgeConfigSpec;
+
+import java.math.BigDecimal;
+
+public class GreaterThanOrEqualPredicate implements IConfigPredicate {
+    private static final ResourceLocation ID = new ResourceLocation(AbnormalsCore.MODID, "greater_than_or_equal_to");
+    private final BigDecimal value;
```

Comment:
I don't see the need to use `BigDecimal`. `Double` would work fine.

Reply:
hmm i guess no one will need that much precision...

Reply:
Added